tests/test_mappings/test_tasks.py

```
@@ -1,4 +1,6 @@
-from apps.mappings.tasks import sync_sage300_attributes
+from fyle_accounting_mappings.models import MappingSetting
+from fyle_integrations_imports.models import ImportLog
```

Ruff (0.8.2)

`fyle_integrations_imports.models.ImportLog` imported but unused

Remove unused import: `fyle_integrations_imports.models.ImportLog`

(F401)